content/unsupported-browser.html

```diff
@@ -0,0 +1,65 @@
+---
+layout: default
+title: Unsupported Browser
+description: Scratch Addons doesn't support the browser you're using.
+---
+
+<section id="intro" class="intro-heading">
+  <h1 class="mb-3">Unsupported browser</h1>
+  <p class="mb-1">
+    Scratch Addons doesn't support the browser you're using.
+  </p>
+</section>
+
+<section class="container">
+  <p>Use updated versions of Chrome, Firefox, or other browsers based on them to get started:</p>
+  <br>
+  <h4>
+    <span class="iconify" data-icon="simple-icons:googlechrome"></span> Chromium 80+
+  </h4>
+  <p>
+    <p>
+      <span class="iconify" data-icon="simple-icons:googlechrome"></span> Chrome 80+
+      &nbsp;<a href="https://google.com/chrome/" target="_blank">Download</a>
```

Comment:
I think `margin-left` (using either a class or inline CSS) is better than using `&nbsp;` for this.

Reply:
Or using ul-li lists